src/holofuel/pages/Home/Home.js

```diff
@@ -45,7 +51,15 @@ export default function Home () {
   const history = useHistory()
   const goToOfferRequest = () => history.push(OFFER_REQUEST_PATH)
 
-  const { data: { holofuelLedger: { balance: holofuelBalance } = { balance: 0 } } = {} } = useQuery(HolofuelLedgerQuery, { fetchPolicy: 'network-only' })
+  const { newMessage } = useFlashMessageContext()
+
+  const filterActionableTransactionsByStatus = useCallback(status => holofuelActionableTransactions.filter(actionableTx => actionableTx.status === status), [holofuelActionableTransactions])
+
+  useEffect(() => {
+    if (!isEmpty(filterActionableTransactionsByStatus('declined'))) {
+      newMessage(declinedTransactionNotice)
```

Comment:
I'm not a fan of this UX, but let's call this the MVP and bring it the UXers for implementation in a new ticket.

Reply:
Yes, this is exactly what this is. :100:   I actually spoke to Zoe and Rob about this on Friday, and there was a interim decision made that is now reflected in Figma (basically a modal (instead of a flash  message) that triggers a bulk action to refund any/all outstanding transactions).  We can definitely PL this more in standup tomorrow though, as you suggested.